hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/nio/NioAsyncSocket.java

```diff
@@ -427,7 +428,7 @@ public boolean unsafeWriteAndFlush(IOBuffer buf) {
         boolean result;
         if (currentFlushThread == null) {
             if (flushThread.compareAndSet(null, currentThread)) {
-                eventloop.localRunQueue.add(eventLoopHandler);
+                eventloop.localQueue.add(eventLoopHandler);
```

Comment:
I'm not sure about exposing all fields. It's convenient during development, but long-term it's probably better to add domain-specific functions like `addLocalTask` with javaDoc of which task is considered local; same for other types.

Reply:
As long as it isn't needed, it won't be added.